test/fail_compilation/staticarrayoverflow2.d

```diff
@@ -0,0 +1,10 @@
+/*
+REQUIRED_ARGS: -m64

```

Comment:
@jacob-carlborg Compiling this test in 32bit mode results in back-end catching the overflow rather than front-end. Changing `long[size_t.max] a;` to `byte[size_t.max] a;` results in `Internal error: backend\dt.c 357`. `byte[size_t.max + 1] a;` gives an access violation error when dmd is compiled in debug mode. Same code compiles successfully, *giving a zero-length array* when dmd is compiled in release mode. This is no different than walking in minefield. I don't want to deal with this anymore.